apartment_application_service/models.py

```diff
@@ -47,13 +48,10 @@ def right_of_residence_ordering_number(self) -> Optional[int]:
         number to the new batch's numbers to make sure that happens. This also maintain
         inner ordering inside both of the batches.
         """
-        if self.right_of_residence is None:
-            return None
```

Comment:
Either leave this here or move it to the function, since now None is no longer handled.